tests_thien/pages/dashboard/dashboard_page.py

```diff
@@ -0,0 +1,73 @@
+from selenpy.element.base_element import BaseElement
+from selenpy.support import browser
+from selenpy.support.conditions import have
+from selenpy.element.text_box import TextBox
+from selenpy.element.combo_box import ComboBox
+from tests_thien.utilities import constant, browser_helper
+from tests_thien.utilities.settings_list import SettingsList
+from tests_thien.utilities.menu_list import MenuList
+
+class DashboardPage():
+    
+    def __init__(self):
+        self._btn_global_setting = BaseElement("class=mn-setting")
+        self._txt_page_name = TextBox("id=name")
+        self._cbb_parent_name = ComboBox("id=parent")
+        self._btn_ok = BaseElement("id=OK")
+        self._lbl_test_module_execution = BaseElement("//div[@title='Test Module Execution']")
+        self._link_page_item= BaseElement("//div[@id='main-menu']//a[text()='%s']")
+        self._item_menu = BaseElement("//a[text()='%s']")
+    
+    def page_title(self,title):
```

Comment:
What's it?

Reply:
function return the page_title that we use later to check